### ISSUE TITLE
Need additional documentation for UDM Pro SE

### ISSUE THREAD

**Issue by absane** [willswire/unifi-cloudflare-ddns#6]
I found that the instructions provided here work as expected for the USG. However, when I replicated the steps for the UDM Pro SE (UnifFi OS 2.4.8, which is EA), DNS was not updating. To resolve it, I SSH'd into it and ran this command:

`/usr/sbin/inadyn -n -C -f /run/ddns-eth8-inadyn.conf --no-pidfile -l debug`

I found that the HTTP request it was making to Cloudflare was malformed because it was missing a leading folder path:

inadyn[20183]: Sending alias table update to DDNS server: 
```
GET foobar.workers.dev HTTP/1.0
Host: foobar.workers.dev
Authorization: Basic foobar
User-Agent: inadyn/2.8.1 https://github.com/troglobit/inadyn/issues

inadyn[20183]: Successfully sent HTTPS request!
inadyn[20183]: Successfully received HTTPS response (316/8191 bytes)!
inadyn[20183]: DDNS server response: HTTP/1.1 400 Bad Request
Server: cloudflare
Date: Thu, 28 Apr 2022 16:22:43 GMT
Content-Type: text/html
Content-Length: 155
Connection: close
CF-RAY: -

<html>
<head><title>400 Bad Request</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
<hr><center>cloudflare</center>
</body>
</html>
```

Notice the malformed `GET subdomain.domain.com HTTP/1.0`

Long story short, thanks to [this blog post](https://dixonwille.medium.com/custom-dynamic-dns-with-unifi-dream-machine-pro-b664f7c964), I found i needed to append `/nic/update?hostname=%h&myip=%i` at the end of the Server entry in the DDNS configuration. So, if your Server entry is `subdomain.workers.dev`, make it this: `workers.dev/nic/update?hostname=%h&myip=%i`.

**Comment by willswire** [comment]
Investigating these issues - in the meantime, the `README.md` has been updated. Please note I have modified the script to accept routes on `/update?hostname=...` instead of `/nic/update`.
